src/js_modules/chooseProgram/index.jsx

```diff
@@ -45,63 +45,71 @@ function ChooseProgram({ chooseList, handleChoose, setLateModalProps }) {
     });
   }) : [];
 
-  const marketingCourses = marketingCursesList?.length > 0 ? marketingCursesList.filter(
+  const hasNonSaasCourse = chooseList.some(({ cohort }) => !cohort.available_as_saas);
+
+  const marketingCourses = marketingCursesList.filter(
     (item) => !activeSubscriptionCohorts.some(
       (activeCohort) => activeCohort?.all_subscriptions?.some(
         (sb) => sb?.selected_cohort_set?.slug === item?.slug,
       ),
     ) && item?.course_translation?.title,
-  ) : [];
+  );
 
-  const isNotAvailableForMktCourses = activeSubscriptionCohorts.length > 0 && activeSubscriptionCohorts.some(
-    (item) => item?.cohort?.available_as_saas === false,
+  const isNotAvailableForMktCourses = activeSubscriptionCohorts.some(
```

Comment:
The name of this variable is confusing as it's relating cohorts with subscriptions but that is not the actual value